packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -755,9 +755,9 @@ Specifies a set of Groups whose Users are to be included or excluded
 
 Specifies an authentication entry point
 
-| Parameter | Description | Data Type         | Required | Default |
-| --------- | ----------- | ----------------- | -------- | ------- |
-| authType  |             | `ANY` or `RADIUS` | No       |         |
+| Parameter | Description | Data Type                             | Required | Default |
+| --------- | ----------- | ------------------------------------- | -------- | ------- |
+| authType  |             | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |         |
```

Comment:
FYI we usually add `<ApiLifecycle access="ea" />` to annotate a specific feature's current lifecycle. For example - https://github.com/okta/okta-developer-docs/pull/1459/files#diff-c46ef0d1a519db411ccedeacea1fe583589f59a754380a2aa2de208c1d103dc6R747, which looks like this:
![Screen Shot 2020-11-24 at 4 33 08 PM](https://user-images.githubusercontent.com/12566503/100166983-cd460680-2e72-11eb-9b52-647e9d0627d0.png)

Reply:
Thanks for telling me this! I added the annotation. Please take a look and see whether it is correct.